site/troubleshooting.md

```diff
@@ -162,8 +166,8 @@ The columns are as follows:
       heartbeat
     * `established` - TCP connection and corresponding UDP path are up
  * Info - the failure reason for failed and retrying connections, or
-   the data transport method, remote peer name and nickname for
-   pending and established connections
+   the data transport method, security level, remote peer name and
+   nickname for pending and established connections
```

Comment:
> We don't make any reference "plaintext" for unencrypted weaves in the docs. And it doesn't sound right anyway, since we are not transmitting "text".

It has a well defined meaning - I don't think anyone thinks that cryptography only works on ASCII text.

> But for those deployments running w/o a password - no doubt the majority - it is just noise.

How about printing it only if a password has been supplied?